dist/tools/testrunner/testrunner.py

```diff
@@ -38,9 +39,15 @@ def run(testfunc, timeout=10, echo=True, traceback=False):
         if traceback:
             print_tb(sys.exc_info()[2])
         return 1
+    except EOF:
+        print("EOF in expect script")
+        return 1
     finally:
         print("")
-        os.killpg(os.getpgid(child.pid), signal.SIGKILL)
-        child.close()
+        try:
+            os.killpg(os.getpgid(child.pid), signal.SIGKILL)
+        except ProcessLookupError:
+            print("Process lookup failed")
```

Comment:
Maybe change the message to: `Process already stopped` ? Otherwise, one could think there's an error, which is not really the case with SSP.